lib/tasks/job.rake

```diff
@@ -10,6 +10,73 @@ namespace :job do
     Sidekiq::ScheduledSet.new.clear
   end
 
+  task running: :environment do
+    next false if Sidekiq::Stats.new.enqueued.zero?
+    job_class = ENV['JOB_CLASS']
+    job_tenant = ENV['TENANT']
+    running = false
+    Sidekiq::Queue.all.each do |queue|
+      jobs = queue.entries.select do |entry|
+        (!job_class || entry.item['args'].first['job_class'] == job_class) &&
```

Comment:
Maybe refactor this in a lambda/proc/other?

Reply:
Hard to do in the context of task definitions. (especially since they depend on the ENV variables)
We could do with a lambda-generating lambda, but I feel like this would add too much complexity for the benefits.